UnityProject/Assets/Scripts/Player/PlayerHealthData.cs

```diff
@@ -39,6 +39,8 @@ public class RaceHealthData
 	public BloodType BloodType;
 
 	public ImplantProcedure RootImplantProcedure;
+
+	public bool BodyPartsShareSameSkinColor = false;
```

Comment:
why not have that on the body part customisation? instead

Reply:
What about body parts that are shared between all races? for example, hair and facial hair are treated as body parts and are shared between all races; if you want it to share colors with your race's skintone you'll run into a problem where enabling this on the hair causes it to happen across all races.

This is a better solution because at the moment there is no race or use case what-so-ever on US13 or SS13 where a race has some body parts/hair that shares their skin tone while others don't. (Unless you count 18+ servers such as Skyrat where they give you the option to sync your gentiles colors with your skin tone).

Reply:
ok, let me get good example, 
let's say if you want your body markings on lizards to be different colour from your skin tone?

Reply:
Are body markings customizable on /TG/? If yes, then I'll change it so we have rules applied to related body parts only.
If not, then we can start a poll on the server to see if we should keep this solution or not as I do feel like Lizards look amazing with matching colors.

Reply:
I just checked and body markings for lizards do not have customizable colors, just different sprites.

Reply:
Good. Can I have my matching colored lizards now pls 
![image](https://user-images.githubusercontent.com/34368774/114524653-b0557880-9c45-11eb-8c3b-f07403a1fc40.png)

Reply:
> at the moment there is no race or use case

I'd rather have future proofing

Reply:
yeah, Future proofing a better

Reply:
Done and I even got my matching colored lizards.